gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/FlowConfigsV2Resource.java

```diff
@@ -128,11 +132,17 @@ public FlowConfig get(ComplexResourceKey<FlowId, FlowStatusId> key) {
   @ReturnEntity
   @Override
   public CreateKVResponse create(FlowConfig flowConfig) {
-    List<ServiceRequester> requestorList = this.requesterService.findRequesters(this);
+    List<ServiceRequester> requesterList = this.requesterService.findRequesters(this);
     try {
-      String serialized = RequesterService.serialize(requestorList);
+      String serialized = RequesterService.serialize(requesterList);
       flowConfig.getProperties().put(RequesterService.REQUESTER_LIST, serialized);
       LOG.info("Rest requester list is " + serialized);
+      String owningGroup = flowConfig.getProperties().getOrDefault("owning_group", "");
+      if (owningGroup != "") {
```

Comment:
Strings cannot be compared like this in java. This will always return false. Please use String's equals method. or guava's isNullOrEmpty()
I haven't thoroughly reviewed the unit test, but if unit test passes with this, maybe they need some correction too.